ibis_ml/steps/_impute.py

```diff
@@ -14,6 +15,8 @@
 
 
 def _fillna(col, val):
+    if val is None or col.type().is_floating() and math.isnan(val):
```

Comment:
```suggestion
    if val is None or and math.isnan(val):
```
Why is the `is_floating()` check necessary?

Also, I think it would make sense to have separate error for `None` and `NaN` (i.e. be specific in the message, rather than saying it could be either)

Reply:
It could be a string col, the `math.isnan(val)` will throw an error